### ISSUE TITLE
Tamed dragon seems to forget about their enemy quite easily [ICE AND FIRE VERSION 2.1.6]

### ISSUE THREAD

**Issue by kihgy** [Alex-the-666/Ice_and_Fire#3659]
**NO SUGGESTIONS AT ALL**
This is not a repository for suggestions or issues not pertaining to the mod's code or gameplay.

**Describe the bug**
Tamed dragons can forget about the mob that they are fighting

**To Reproduce**
1. Spawn tamed and wild dragon
2. Have them fight
3. Watch as wild dragon is more consistent in targetting tamed dragon

**Expected behavior**
Tamed dragons will just as determined to attack the target as wild dragons

**Screenshots**

**Additional context**

**Comment by warning-djkajslh** [comment]
Yeah it seems that as soon tamed dragons start flying, it will forgot about the target, or when they got too far away. Not happening for wild dragons tho.
